api/apps/api/config/default.json

```diff
@@ -58,12 +58,15 @@
     "apikey": "invalidSparkpostApikey",
     "origin": "https://api.eu.sparkpost.com:443"
   },
+  "application": {
+    "baseUrl": "http://localhost:3000/"
+  },
   "passwordReset": {
-    "tokenPrefix": "http://localhost:3000/auth/reset-password?token=",
+    "tokenPrefix": "auth/reset-password?token=",
     "expiration": 1800000
   },
   "signUpConfirmation": {
-    "tokenPrefix": "http://localhost:3000/auth/sign-up-confirmation?token="
+    "tokenPrefix": "auth/sign-up-confirmation?token="
   }
```

Comment:
nitpicking on this - I would suggest to keep the host part of the URL free from trailing `/` and add it as leading to the path component (or fragments thereof), such as `"tokenPrefix": "/auth/reset-password?token=",` and so on - the current implementation won't cause any trouble, but IIRC we tried to stick to something like this after we had some trouble with external APIs being picky on double-slashes somewhere in the path.

Reply:
Fixed on https://github.com/Vizzuality/marxan-cloud/pull/664/commits/6da3d2e4ae5631a234dcabc2d895beec37adc0e6